moonraker_obico/app.py

```diff
@@ -101,7 +103,7 @@ def wait_for_auth_token(self, args):
             config=config,
             remote_status={'viewing': False, 'should_watch': False},
             linked_printer=linked_printer,
-            printer_state=PrinterState(config),
+            printer_state=PrinterState(config, self),
```

Comment:
PrinterState shouldn't depend on `app`. Think about how to remove this dependency.